### ISSUE TITLE
Master branch submodules not accessible to public

### ISSUE THREAD

**Issue by kaidoho** [pulp-platform/pulpissimo#39]
Hi guys,

cloning the master branch requires a password to pull in the submodules that have been added  a couple of days ago. Is that intended?

```console
blofeld@PULP-Test:~/projects$ git clone --recursive https://github.com/pulp-platform/pulpissimo.git 
Cloning into 'pulpissimo'...
remote: Enumerating objects: 93, done.
remote: Counting objects: 100% (93/93), done.
remote: Compressing objects: 100% (70/70), done.
remote: Total 598 (delta 48), reused 58 (delta 23), pack-reused 505
Receiving objects: 100% (598/598), 3.11 MiB | 2.52 MiB/s, done.
Resolving deltas: 100% (275/275), done.
Submodule 'ml_tests' (git@iis-git.ee.ethz.ch:pulp-tests/ml_tests.git) registered for path 'ml_tests'
Submodule 'parallel_bare_tests' (git@iis-git.ee.ethz.ch:pulp-sw/parallel_bare_tests.git) registered for path 'parallel_bare_tests'
Submodule 'pulp_tests' (git@iis-git.ee.ethz.ch:pulp-sw/pulp_tests.git) registered for path 'pulp_tests'
Submodule 'riscv_tests' (git@iis-git.ee.ethz.ch:pulp-sw/riscv_tests.git) registered for path 'riscv_tests'
Submodule 'rt-tests' (git@iis-git.ee.ethz.ch:pulp-tests/rt-tests.git) registered for path 'rt-tests'
Submodule 'sequential_bare_tests' (git@iis-git.ee.ethz.ch:pulp-sw/sequential_bare_tests.git) registered for path 'sequential_bare_tests'
Cloning into '/home/blofeld/projects/pulpissimo/ml_tests'...
git@iis-git.ee.ethz.ch's password: 
```

**Comment by haugoug** [comment]
You don't need submodules, they are just used for testing purpose, just remove the option --recursive. We'll remove these submodules soon.

**Comment by kaidoho** [comment]
Hi,

thx for the immediate reply guys.

@haugoug that's what I've done and actually what the readme tells me to do. I just saw them and due to the fact that they are on master and not some feature branch I kind of expected to require them. 

Anyway, it's good to talk to you @haugoug. You've moved the SDK doc to pulp-sdk-doc repository. However, the doc folder still exists within pulp-sdk. Thus, git is not able to checkout pulp-sdk-doc and the SDK build fails. It works if I delete the doc folder before building the SDK. 
I can open an issue for that, as well as for a very few other minor things I came across when trying to get Pulp up and running, but I haven't found a dedicated issue tracker for pulp-sdk. Where is the main entry point for issues - pulp or pulpissimo?
